### PR TITLE
Create monitoring bundles

### PR DESCRIPTION
**Pull request by lulf**

### CODE REVIEW

File: documentation/assemblies/assembly-monitoring-kube.adoc [hunk 1]
```diff
@@ -9,10 +9,14 @@
 [id='monitoring-kube-{context}']
 = Monitoring {ProductName} on Kubernetes
 
-You can use Prometheus and Grafana for monitoring the {ProductName} service.
+Monitoring services run frequent health checks  on {ProductName} and send alerts when health checks fail. Health is assessed using Prometheus and kube-state-metrics. Alerting is implemented with Alertmanager. Grafana is also configured to provide a dashboard of the current status of health checks.
```
Comment (jenmalloy) [inline documentation/assemblies/assembly-monitoring-kube.adoc]: `kube-state-metrics`
`Alertmanager`

File: documentation/assemblies/assembly-monitoring-oc.adoc [hunk 2]
```diff
@@ -9,10 +9,14 @@
 [id='monitoring-oc-{context}']
 = Monitoring {ProductName} on OpenShift
 
-You can use Prometheus and Grafana for monitoring the {ProductName} service.
+Monitoring services run frequent health checks  on {ProductName} and send alerts when health checks fail. Health is assessed using Prometheus and kube-state-metrics. Alerting is implemented with Alertmanager. Grafana is also configured to provide a dashboard of the current status of health checks.
```
Comment (jenmalloy) [inline documentation/assemblies/assembly-monitoring-oc.adoc]: `kube-state-metrics`
`Alertmanager`

File: documentation/modules/proc-deploy-alertmanager.adoc [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+//
+// assembly-monitoring-kube.adoc
+// assembly-monitoring-oc.adoc
+
+[id='deploy-alertmanager-{context}']
+= Deploying Alertmanager
+
+.Prerequisites
+
+* You must have already installed Prometheus and kube-state-metrics.
```
Comment (jenmalloy) [inline documentation/modules/proc-deploy-alertmanager.adoc]: `kube-state-metrics`

File: documentation/modules/proc-deploy-alertmanager.adoc [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+.Procedure
+
+. Replace the namespace with the namespace EnMasse is currently deployed to:
```
Comment (jenmalloy) [inline documentation/modules/proc-deploy-alertmanager.adoc]: Change "EnMasse" to "{ProductName}"

File: documentation/modules/proc-deploy-grafana.adoc [hunk 5]
```diff
@@ -6,8 +6,19 @@
 [id='deploy-grafana-{context}']
 = Deploying Grafana
 
+.Prerequisites
+
+* You must have already installed Prometheus and kube-state-metrics.
+
 .Procedure
 
+. Replace the namespace with the namespace EnMasse is currently deployed to:
```
Comment (jenmalloy) [inline documentation/modules/proc-deploy-grafana.adoc]: Change "EnMasse" to "{ProductName}"

File: documentation/modules/proc-deploy-kube-state-metrics.adoc [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+//
+// assembly-monitoring-kube.adoc
+// assembly-monitoring-oc.adoc
+
+[id='deploy-kube-state-metrics-{context}']
+= Deploying kube-state-metrics
+
+.Procedure
+
+. Replace the namespace with the namespace {ProductName} is currently deployed to:
```
Comment (jenmalloy) [inline documentation/modules/proc-deploy-kube-state-metrics.adoc]: Change "EnMasse" to "{ProductName}"

File: documentation/modules/proc-deploy-prometheus.adoc [hunk 7]
```diff
@@ -8,6 +8,13 @@
 
 .Procedure
 
+. Replace the namespace with the namespace EnMasse is currently deployed to:
```
Comment (jenmalloy) [inline documentation/modules/proc-deploy-prometheus.adoc]: Change "EnMasse" to "{ProductName}"

File: documentation/modules/proc-deploy-kube-state-metrics.adoc [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
++
+[options="nowrap",subs="attributes"]
+----
+sed -i 's/{ProductNamespace}/_my-namespace_/' install/components/kube-state-metrics/*.yaml
+----
+
+. Create the kube-state-metrics deployment:
```
Comment (jenmalloy) [inline documentation/modules/proc-deploy-kube-state-metrics.adoc]: `kube-state-metrics`

File: documentation/modules/proc-deploy-grafana.adoc [hunk 9]
```diff
@@ -6,8 +6,19 @@
 [id='deploy-grafana-{context}']
 = Deploying Grafana
 
+.Prerequisites
+
+* You must have already installed Prometheus and kube-state-metrics.
```
Comment (jenmalloy) [inline documentation/modules/proc-deploy-grafana.adoc]: `kube-state-metrics`

File: documentation/modules/proc-deploy-alertmanager.adoc [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
++
+[options="nowrap",subs="attributes"]
+----
+sed -i 's/{ProductNamespace}/_my-namespace_/' install/components/alertmanager/*.yaml
+----
+
+. Create the Alertmanager deployment:
```
Comment (jenmalloy) [inline documentation/modules/proc-deploy-alertmanager.adoc]: `Alertmanager`

File: documentation/modules/proc-deploy-alertmanager.adoc [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} apply -f ./install/components/alertmanager
+----
+
+. (Optional) Configure Alertmanager to send emails using a custom configmap with the relevant SMTP details
```
Comment (jenmalloy) [inline documentation/modules/proc-deploy-alertmanager.adoc]: `Alertmanager`
details:
(add a colon^)

File: documentation/modules/proc-deploy-alertmanager.adoc [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} apply -f ./alertmanager-configmap.yaml
+----
+An example Alertmanager config map:
```
Comment (jenmalloy) [inline documentation/modules/proc-deploy-alertmanager.adoc]: `Alertmanager`

File: documentation/modules/proc-installing-using-ansible.adoc [hunk 13]
```diff
@@ -29,6 +29,12 @@ The following Ansible configuration settings are supported:
 |api_server|Specifies whether to enable the REST API server.|True |no
 |register_api_server|Specifies whether to register the API server with Kubernetes master.|False |no
 |secure_api_server|Specifies whether to enable mutual TLS for the API server.|False |no
+|enable_monitoring|Specifies whether to enable the monitoring services|Not applicable|no
+|smtp_server|Specifies the SMTP server used to send alert emails|Not applicable|no
+|smtp_username|Specifies the username used to authenticate to the SMTP server|Not applicable|no
+|smtp_password|Specifies the password used to authenticate to the SMTP server|Not applicable|no
+|smtp_from_address|Specifies the from address displayed in alert emails|Not applicable|no
+|sysadmin_email|Specifies the email address alerts should be sent to|Not applicable|no
```
Comment (jenmalloy) [inline documentation/modules/proc-installing-using-ansible.adoc]: Specifies the email address to which alerts are sent

**Comment by lulf** [comment]
@jenmalloy Additional comments addressed

**Pull request merged**